app/views/courses/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= content_for :page_title, "#{@course.name} (#{@course.course_code}) - Courses" %>
+
+<%= link_to 'Back', provider_courses_path, class: "govuk-back-link" %>
+
+<main class="govuk-main-wrapper " id="main-content" role="main">
```

Comment:
We reuse this bit in a few places now?

Reply:
Yes, I have an idea on how to dry this up :)